packages/react/src/components/controlComponents.tsx

```diff
@@ -169,6 +169,7 @@ export const RedirectToSignUp = withClerk(({ clerk, ...props }: WithClerkProp<Re
 }, 'RedirectToSignUp');
 
 /**
+ * @function
```

Comment:
Typedoc is now more strict about `const foo = () => {}` vs `function bar() {}`.

Without the `@function` keyword it marks the former as a variable, not a function. Since I didn't want to change the function signature everywhere (since that should be enforced with ESLint) I just added the annotations